src/deploy/functions/build.ts

```diff
@@ -494,10 +494,12 @@ export function toBackend(
         nullsafeVisitor((cpu) => (cpu === "gcf_gen1" ? cpu : r.resolveInt(cpu)))
       );
       if (bdEndpoint.vpc) {
+        bdEndpoint.vpc = { connector: params.resolveString(bdEndpoint.vpc.connector, paramValues) };
```

Comment:
You are writing over the entire `vpc` object which will remove the field `egressSettings` if it exists

Reply:
```
bdEndpoint.vpc.connector = params.resolveString(bdEndpoint.vpc.connector, paramValues);
```

Reply:
ok committed that fix.